dangerfile.js

```diff
@@ -108,10 +108,10 @@ if (allFiles.length > 100) {
   allFiles.forEach(file => {
     // eslint-disable-next-line promise/prefer-await-to-then
     danger.git.diffForFile(file).then(({ diff }) => {
-      if (/serverSecrets/.test(diff) && !secretsDocs.modified) {
+      if (/authHelper/.test(diff) && !secretsDocs.modified) {
```

Comment:
Should we check for both of these for now, and then remove `serverSecrets` once all the services have been updated to work with the new model?